docs/howtos/DataPlane Generation - DPG/03convenient.mdx

```diff
@@ -235,7 +255,20 @@ FIXME
 <TabItem value="java" label="Java" >
 
 ```java
-FIXME
+package com.azure.ai.openai.models;
+
+public final class AzureCognitiveSearchIndexFieldMappingOptions {
+    public AzureCognitiveSearchIndexFieldMappingOptions()
```

Comment:
What is the difference with not having usage? mention using comments?

Reply:
usually the public/private of the ctor.

If it is output only, its ctor is private.